dot/parachain/prospective-parachains/prospective-parachains.go

```diff
@@ -57,7 +68,7 @@ func (pp *ProspectiveParachains) processMessage(msg any) {
 	case CandidateBacked:
 		panic("not implemented yet: see issue #4309")
 	case GetBackableCandidates:
-		panic("not implemented yet: see issue #4310")
+		go pp.getBackableCandidates(msg)
```

Comment:
If we run this in a goroutine, access to `pp.View` needs to be thread-safe. Did the design suggest to do this? The method shouldn't take very long to execute. 🤔